### PR TITLE
Send to Graphite

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: handlers.go [hunk 1]
```diff
@@ -4,27 +4,51 @@ import (
...
+	latestWrite := make(chan fthealth.HealthResult)
+	hcPeriod := make(chan time.Duration)
+	hch := &hchandlers{registry, checker, "Coco Aggregate Healthcheck", "Checks the health of all deployed services",
+		latestRead, graphiteFeeder, kapi, hcPeriod}
+
+	// set up channels for buffering data to be sent to Graphite
+	latestGraphiteWrite := make(chan *HealthTimed)
+	latestGraphiteRead := make(chan *HealthTimed, 10)
+	ring := NewRingBuffer(latestGraphiteWrite, latestGraphiteRead)
+	go ring.Run()
```
Comment (martingartonft) [inline handlers.go]: Why is this ringbuffer needed?  Why not just a single buffered channel?

Comment (kovacshuni) [inline handlers.go]: Removed ringbuffer.

**Comment by tamas-molnar** [comment]
lgtm

**Comment by Nesze** [comment]
+1

**Pull request merged**